src/test/osd/osd-scrub-repair.sh

```diff
@@ -87,6 +92,202 @@ function TEST_corrupt_and_repair_replicated() {
     teardown $dir || return 1
 }
 
+# the utility function to mess up with shard file without starting/stopping
+# the involved OSD, thus faster =)
+# dir, osd, obj, pool, op, args...
+function faster_objectstore_tool()
+{
+    local dir=$1
+    shift
+    local osd=$1
+    shift
+    local obj=$1
+    shift
+    local pool=$1
+    shift
+    local op=$1
+    shift
+
+    local pg=$(get_pg $pool $obj)
+    local shard
+    if ceph osd pool get $pool erasure_code_profile 2>/dev/null; then
+        # ec pool
+        local shard_id=`basename $osd`
+        osd=`dirname $osd`
+        shard=($dir/$osd/current/${pg}s${shard_id}_head/${obj}__head*)
+    else
+        # replicated pool
+        shard=($dir/$osd/current/${pg}_head/${obj}__head*)
+    fi
+    echo $replica
+    case $op in
+        set-bytes)
+            # write up to the size of $shard
+            dd if=$1 of=$shard bs=1 count=$(stat --format '%s' $shard) conv=notrunc
+            ;;
+        truncate)
+            dd if=$1 of=$shard
+            ;;
+        remove)
+            rm $shard
+            ;;
+        get)
+            cp $shard $1
+            ;;
+        path)
+            echo $shard
+    esac
+}
+
+#
+# Corrupt one copy of a replicated pool and repair it manually
+#
+# TODO: move this test to ceph-qa-suite
```

Comment:
if this test is moved to `qa/suites/rados/basic/tasks`, i think it will have the coverage of the various qa-suite options.

maybe we can postpone this to another PR? i'd really want to have this feature in Luminous if possible.